### DISCUSSION TITLE
编译型语言

### DISCUSSION THREAD

**Discussion by cyb233** [InvoluteHell/ThreeKeysProgramming#16] [Ideas]
将像关键字之类以及类型声明等一些无可奈何的东西放到规则之外？
比如以Java为例的
```java
public static class a {
    public static int a;
    public static void a(){
        return a;
    }
}
```
这里的public，static，class，return，void是关键字，int是类型声明

不过光Java一个System.out.println()就12个字母了，难办哦

**Comment by MistEO** [comment]
关键字那就太广了，似乎可以考虑程序入口部分不纳入字母统计

**Reply by MistEO** [reply]
入口函数放开了: https://github.com/InvoluteHell/ThreeKeysProgramming/pull/22
